examples/algorithms/plot_greedy_coloring.py

```diff
@@ -0,0 +1,40 @@
+"""
+=====================
+Greedy Coloring Algorithm
+=====================
+We attempt to color a graph using as few colors as possible, where no neighbours of a node can have same color as the node itself.We use the function nx.greedy_color that implements the greedy coloring algorithm for a given graph.The function returns a dictionary that maps each node to its assigned color.
+"""
+import networkx as nx
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+
+# Create a dodecahedral graph
+G = nx.generators.small.dodecahedral_graph()
```

Comment:
We generally recommend calling functions from the main namespace when they're available:

```suggestion
G = nx.dodecahedral_graph()
```

Reply:
Noted! Will make sure to follow this practice now.